internal/util/kubestructs.go

```diff
@@ -0,0 +1,39 @@
+package util
+
+import (
+	"fmt"
+
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+func NewTestingCRD(name, group string, unique bool, versions []apiextensionsv1.CustomResourceDefinitionVersion) *apiextensionsv1.CustomResourceDefinition {
```

Comment:
This function signature is a little funny. WDYT about something like
```
if name == "" {
  name = GenName(defaultName)
} 
```
and get rid of the unique bool altogether?